install_config/configuring_routing.adoc

```diff
@@ -16,6 +16,8 @@ xref:../install_config/router/index.adoc#install-config-router-overview[deployin
 a router], you can configure the default timeouts for an existing route when you
 have services in need of a low timeout, as required for Service Level
 Availability (SLA) purposes, or a high timeout, for cases with a slow back end.
+If undefined, the default route timeout is defined by ROUTER_DEFAULT_SERVER_TIMEOUT
```

Comment:
Suggested edit + correcting link:

If undefined, the default route timeout is defined by xref:../architecture/networking/routes.adoc#env-variables[`ROUTER_DEFAULT_SERVER_TIMEOUT`].